foundry.toml

```diff
@@ -3,6 +3,7 @@ src = "src"
 out = "out"
 test = "test"
 libs = ["lib"]
+optimizer_runs = 999999 # Etherscan does not support verifying contracts with more optimization runs.
```

Comment:
really? It's weird no?

Reply:
I believe it is to avoid timing out when compiling contracts
Not so surprising neither considering 999999 is already gigantic and more optimization runs are not significant